IBM/CAST
# fixes for broken ufm APIs and ufm inventory collection

# Purpose
new versions of ufm restAPIs broke csm apis and csm inventory collection. this pull request addresses those issues so that functionality is restored. 

# Approach
- still in progress of checking ufm updates, but i noticed these and pushed the fixes. 
- still checking more ufm update issues.
- working on notes in case we need to help the labs with the update

# Origin
https://github.com/IBM/CAST/issues/613

# How to Test
1. re run regression - make sure the apis still run
2. update ufm drivers
3. run inventory collection
4. see expected results

#Reviewers
- [x] @williammorrison2  for database
- [ ] @mew2057  for code
- [ ] @pdlun92  for test
